src/SFA.DAS.AssessorService.Application.Api.UnitTests/Validators/Register/EpaORganisationSearchValidatorTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using NUnit.Framework;
+using SFA.DAS.AssessorService.Application.Api.Validators;
+
+namespace SFA.DAS.AssessorService.Application.Api.UnitTests.Validators.Register
+{
+    [TestFixture]
+    public class EpaORganisationSearchValidatorTests
+    {
+        private EpaOrganisationSearchValidator _validator;
+
+        [SetUp]
+        public void Setup()
+        {
+            _validator = new EpaOrganisationSearchValidator();
+        }
+
+        [TestCase("EPA0000", true, null)]
+        [TestCase("EPA000", false, "checking there is at least 4 numbers after the 'epa'")]
+        [TestCase("epa0000", true, null)]
+        [TestCase("EPA0000A", false, null)]
+        [TestCase("EPA000000000", true, null)]
+        [TestCase("EPA0000000000", false, "checking it is not greated that 12 characters in all")]
+        [TestCase("stuff", false, null)]
+        [TestCase("", false, null)]
+        public void CheckEpaOrganisationIdIsAcceptable(string organisationIdToCheck, bool matching, string message)
+        {
+            var res = _validator.IsValidEpaOrganisationId(organisationIdToCheck);
```

Comment:
Not sure about the use of the word res ? if it;s a result maybe should be called result

Reply:
I was being lazy, will update.

Reply:
PS. I've done this all over the place, so there'll be some new lines of changed code dragged in 🗡

Reply:
Well, I'm a strong believer in the Boy Scout Rule... :p